docs/modules/ROOT/pages/getting-started/filters-and-projections.adoc

```diff
@@ -0,0 +1,191 @@
+= Filters and projections
+
+After making a basic `MATCH` & `RETURN` Cypher query in xref:getting-started/your-first-query.adoc[Your First Query], in this guide you'll add some filters and change the projection to only return some fields.
```

Comment:
Sorry, for some reason I skipped this file during my review, so I'm catching up now.

```suggestion
This tutorial shows how to add filters and change the projection to return specific fields. 
Refer to xref:getting-started/your-first-query.adoc[Your First Query] for more information on how to write queries.
```

^ I made a suggestion about the name of the page Your First Query, but I kept it the way it is right now so we can check later.

Reply:
It would be good to refer here that params are also being addressed in this page. I'm not sure how to suggest that though.